src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -70,28 +70,22 @@ internal sealed class EslintBridgeAnalyzer : IEslintBridgeAnalyzer
         }
 
         public async Task<IReadOnlyCollection<IAnalysisIssue>> Analyze(string filePath, string tsConfig, CancellationToken cancellationToken)
-        {
-            try
-            {
-                return await GetFileIssues(filePath, tsConfig, cancellationToken);
-            }
-            catch (EslintBridgeClientNotInitializedException)
-            {
-                RequireLinterUpdate();
-                return await GetFileIssues(filePath, tsConfig, cancellationToken);
-            }
-        }
-
-        private async Task<IReadOnlyCollection<IAnalysisIssue>> GetFileIssues(string filePath, string tsConfig, CancellationToken cancellationToken)
         {
             await EnsureEslintBridgeClientIsInitialized(rulesProvider.GetActiveRulesConfiguration(), cancellationToken);
-
             var analysisResponse = await eslintBridgeClient.Analyze(filePath, tsConfig, cancellationToken);
 
             if (analysisResponse.ParsingError != null)
             {
-                LogParsingError(filePath, analysisResponse.ParsingError);
-                return Array.Empty<IAnalysisIssue>();
+                if (analysisResponse.ParsingError.Message.Contains(Resources.ERR_ParsingError_LinterNotInitialized))
```

Comment:
This is now the bug fix: I decided to leave the decision up to eslint-bridge itself. If it returns an error that init-linter should be called, I call it.

Reply:
Perhaps we should ask the JS bubble to return a specific error code so we don't have to parse the string.

Reply:
Yep, that could be great. I'll ask Tibor about the localization.